packages/@uppy/webcam/src/index.js

```diff
@@ -168,9 +168,12 @@ module.exports = class Webcam extends Plugin {
       this.opts.modes.indexOf('video-only') !== -1 ||
       this.opts.modes.indexOf('picture') !== -1
 
+    const videoConstraints = this.opts.videoConstraints ?? {}
+    videoConstraints.facingMode = this.opts.facingMode
```

Comment:
Reading this again it should prob prefer the `videoConstraints.facingMode` value over `facingMode` if the former was already set